src/hammer-vlsi/defaults.yml

```diff
@@ -970,6 +978,39 @@ power.submit:
   settings: "vlsi.submit.settings"
   settings_meta: lazycrossref
 
+# Specific inputs for the formal tool.
+# These may vary based on the specific tool plugin
```

Comment:
Can this comment be made a bit clearer in terms of what is varying between plugins?